contrib/pkg/createcluster/create.go

```diff
@@ -249,7 +252,12 @@ func (o *Options) Validate(cmd *cobra.Command) error {
 type cloudProvider interface {
 	generateCredentialsSecret(o *Options) (*corev1.Secret, error)
 
-	addPlatformDetails(o *Options, cd *hivev1.ClusterDeployment) error
+	addPlatformDetails(
+		o *Options,
+		cd *hivev1.ClusterDeployment,
+		installConfig *installertypes.InstallConfig,
+		controlPlanePool,
```

Comment:
Why are the machine pools passed in separately rather than using the fields from the install config?

Reply:
Ugh I have absolutely no idea, will adjust, didn't even see that.